### PR TITLE
Cockpit 173

### PR DESCRIPTION
**Pull request by martinpitt**
Rendered preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-173.html

### CODE REVIEW

File: _posts/2018-07-25-cockpit-173.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### Storage: Offer installation of VDO
+
+The Storage page now shows an "Install VDO support" link if the
+[VDO](https://github.com/dm-vdo/vdo) package is not installed. This is similar
+to the on-demand installation of the NFS client packages in
+[Cockpit 169](https://cockpit-project.org/blog/cockpit-169.html).
+
+![Install VDO on demand](/images/vdo-install-on-demand.png)
+
+### Machines: Add disks to virtual machine
```
Comment (allisonkarlitskaya) [inline _posts/2018-07-25-cockpit-173.md]: I'd write either "to a virtual machine" or (better) "to virtual machines".

Comment (martinpitt) [reply _posts/2018-07-25-cockpit-173.md]: Thanks! Fixed.

File: _posts/2018-07-25-cockpit-173.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+to the on-demand installation of the NFS client packages in
+[Cockpit 169](https://cockpit-project.org/blog/cockpit-169.html).
+
+![Install VDO on demand](/images/vdo-install-on-demand.png)
+
+### Machines: Add disks to a virtual machine
+
+The "Disks" tab of a virtual machine now has a new "Add Disk" button. This can
+create a new virtual disk in a [libvirt storage
+pool](https://libvirt.org/storage.html) or attach an existing one.
```
Comment (garrett) [inline _posts/2018-07-25-cockpit-173.md]: Perhaps something like this?

> The "Disks" tab of a virtual machine now has an "Add Disk" button. Using this new feature, it's possible to create a virtual disk in a [libvirt storage pool](https://libvirt.org/storage.html) or attach an existing disk.

Comment (martinpitt) [reply _posts/2018-07-25-cockpit-173.md]: Thanks! Pushed.

**Pull request merged**